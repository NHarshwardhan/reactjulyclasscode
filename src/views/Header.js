import React from 'react'

export default function Header({title, subtitle}) {
  return (
    <div>
        <h1>{title}</h1>
         
    </div>
  )
}
