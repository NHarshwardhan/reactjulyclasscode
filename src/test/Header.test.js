
import renderer from 'react-test-renderer'
import Header from '../views/Header'

test('render correctly',()=>{

   const tree = renderer.create(<Header title='Welcome To My App'/>).toJSON()
   expect(tree).toMatchSnapshot()

})