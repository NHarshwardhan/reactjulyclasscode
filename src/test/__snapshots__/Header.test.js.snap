// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly 1`] = `
<div>
  <h1>
    Welcome To My App
  </h1>
</div>
`;
